src/main/resources/static/javascript/mainApp/groupings.factory.service.js

```diff
@@ -4,11 +4,11 @@
 
     /**
      * Service for requesting data from the groupings API.
-     * @name groupingsService
+     * @name groupingsFactoryService
      * NOTE: All API requests defined here require at least two parameters, namely onSuccess and onError, which are
      * controller handlers(callback functions) for manipulating data on a successful or unsuccessful request.
      */
-    UHGroupingsApp.factory("groupingsService", function (dataProvider, BASE_URL) {
+    UHGroupingsApp.factory("groupingsFactoryService", function (dataProvider, BASE_URL) {
```

Comment:
I think it would be best to keep this file the same as before but also replace `.factory` with `.service` on this highlighted line. Calling it service makes much more sense because this file does not resemble the factory design pattern.